src/translations/sqlb_de.ts

```diff
@@ -3688,27 +3688,27 @@ Sie können SQL-Statements aus der Schemaspalte nehmen und in den SQL-Editor ode
     <message>
         <location filename="../MainWindow.ui" line="1502"/>
         <source>&amp;DB Toolbar</source>
-        <translation>&amp;DB Toolbar</translation>
+        <translation>&amp;DB Werkzeugleiste</translation>
     </message>
     <message>
         <location filename="../MainWindow.ui" line="1505"/>
         <source>Shows or hides the Database toolbar.</source>
-        <translation>Zeigt oder versteckt die Datenbank-Toolbar.</translation>
+        <translation>Zeigt oder versteckt die Datenbank-Werkzeugleiste.</translation>
     </message>
     <message>
         <location filename="../MainWindow.ui" line="1520"/>
         <source>Shift+F1</source>
-        <translation></translation>
+        <translation>Umschalt+F1</translation>
     </message>
     <message>
         <location filename="../MainWindow.ui" line="1536"/>
         <source>&amp;Recently opened</source>
-        <translation>&amp;Kürzlich geöffnet</translation>
+        <translation>&amp;Zuletzt geöffnet</translation>
```

Comment:
This replaces the keyboard shortcut. Have you verified that there ideally are no duplicates?